src/Fp/Collections/SeqTerminalOps.php

```diff
@@ -308,4 +308,29 @@ public function max(): Option;
      * @psalm-return Option<TV>
      */
     public function maxBy(callable $callback): Option;
+
+    /**
+     * Returns the minimum value from collection
+     *
+     * ```php
+     * >>> LinkedList::collect([1, 4, 2])->min()->get();
+     * => 1
+     * ```
+     *
+     * @psalm-return Option<TV>
+     */
+    public function min(): Option;
+
+    /**
+     * Returns the minimum value from collection by iterating each element using the callback
+     *
+     * ```php
+     * >>> LinkedList::collect([new Foo(1), new Bar(6), new Foo(2)])->minBy(fn(Foo $foo) => $foo->a)->get();
+     * => 1
```

Comment:
Is it `Foo(1)` and not `1` in the example?

Reply:
ok, sorry, fixed it